README.md

```diff
@@ -72,159 +72,159 @@ Identity Providers are responsible for validating the identity of Swift Citizens
+Both Delegated Nodes and Identity Providers receive a salary in Swifts for their service. Salary will be paid out on a daily basis at the same time Swift Citizens receive their daily income. The Swifts generated will initially be drawn from the 80% pool dedicated to regular Swift distribution. Once that pool has been fully distributed, Swifts for salaries will be created in addition to the Stage Two inflation distribution.
 
 ### Delegated Node Salary
-Delegated Nodes will each receive an equal salary. The Delegated Nodes themselves will vote on their own salary with the constraint that the salary must stay within a certain percentage of the previous salary. In the event where Delegated Nodes attempt to abuse this power, it is the responsibility of Swift Citizens to vote out the nodes.
+Delegated Nodes will each receive an equal salary. The Delegated Nodes themselves will vote on their own salary--with the constraint that the salary must stay within a certain percentage of the previous salary. In the event that Delegated Nodes attempt to abuse this power, it's the responsibility of Swift Citizens to [vote out the nodes](#elections).
 
 ### Identity Provider Salary
-The Identity Provider salary is decided on by the delegated nodes. The salary is multiplied by the amount of Swifts that have been claimed by Citizens that are under that specific Identity Provider. For example if the salary decided by the Delegated Nodes is 0.01 and the Identity Provider has 100,000 active validated citizens that claimed 5,000,000 Swifts during that day the Identity Provider will receive 50,000 Swifts at the first block after midnight UTC. 
+The _Salary Multiplier_ for Identity Providers is decided by the Delegated Nodes. An Identity Provider's salary is determined multiplying by the number of Swifts that have been claimed by Citizens under that specific Identity Provider by the Salary Multiplier. For example, if the Salary Multiplier is 0.01 and the Identity Provider has 100,000 active validated citizens that claimed 5,000,000 Swifts during that day, the Identity Provider will receive 50,000 Swifts at the first block after midnight UTC. 
 
 ## Reserved Funds
-20 Billion Swifts will be reserved, these Swifts are not controlled by any central source, but rather are under the control of the Delegated Nodes. It is the responsibility of the Delegated Nodes to assign these funds on an as needed basis to Identity Providers, Delegated Nodes, Swift Citizens, or Swift Accounts in an effort to further the success of the Swift Protocol.
+20 Billion Swifts will be reserved; these Swifts are not controlled by any central source, but are under the control of the Delegated Nodes. It's the responsibility of the Delegated Nodes to assign these funds on an as-needed basis to Identity Providers, Delegated Nodes, Swift Citizens, or Swift Accounts in an effort to further the success of the Swift Protocol.
```

Comment:
The but in `central source, but are under` is a bit strange. This sentence can probably just be reworded.

Reply:
Sounds good.  Watch the comma splice there and "but rather" is redundant.